README.md

```diff
@@ -3,3 +3,8 @@
 Build/tool image with Go 1.12, librkafaka and build tools (git, gcc, make...) installed and ready to run.
 
 Useful to run tests in CI and as base image for multi-stage builds.
+
+There are two images available, based on Alpine (default) and Debian stretch.
+
+If you plan to run tests with `ginkgo` and race detection enabled there's an issue with [Alpine](https://groups.google.com/forum/#!topic/golang-nuts/WnfKCoaRP_E).
+The Debian stretch must be used for this use case.
```

Comment:
It works like that, but if it works better with Debian, why wouldn’t we want to use it as the default one?